src/EmbeddedTypefaceFinder.cpp

```diff
@@ -42,22 +44,21 @@ OsmAnd::EmbeddedTypefaceFinder::EmbeddedTypefaceFinder(
                 new QByteArray(typefaceData),
                 [](void* pUserData) { delete reinterpret_cast<QByteArray*>(pUserData); }),
             [](auto p) { hb_blob_destroy(p); });
-        if (!hbBlob) {
+        if (!hbBlob)
+        {
             LogPrintf(LogSeverityLevel::Error,
                 "Failed to load Harfbuzz blob from embedded data for '%s'",
                 qPrintable(embeddedTypefaceResource));
             continue;
         }
 
-        const auto pHbTypeface = hb_face_create(hbBlob.get(), 0);
-        if (!pHbTypeface || pHbTypeface == hb_face_get_empty()) {
-            LogPrintf(LogSeverityLevel::Error,
-                "Failed to create Harfbuzz typeface from embedded data for '%s'",
-                qPrintable(embeddedTypefaceResource));
+        auto newItem = OsmAnd::ITypefaceFinder::constructTypeface(skTypeface, hbBlob);
```

Comment:
Why do you pass hb_blob instead of hb_face?

Reply:
`hb_blob` uses for creation of the `hb_face`, this functionality is inside `OsmAnd::ITypefaceFinder::constructTypeface(...)`
https://github.com/osmandapp/OsmAnd-core/blob/8865e643a17a29485c2ead3b8be7f65721327b03/src/ITypefaceFinder.cpp#L45

Reply:
Why not to pass hb_face as it was done before? In theory, hb_face may originate elsewhere than from hb_blob

Reply:
Just because of it is a duplicated code according to current state of the code. When (or even IF) it will be required e could change it according to the future case.

Reply:
This is unneeded change, IMHO.

Reply:
Also, pushing this forward - you can bass QByteArray and create hb_blob internally :) To avoid code duplication